src/util/charmrun-src/charmrun.C

```diff
@@ -4446,7 +4451,7 @@ static void finish_nodes(std::vector<nodetab_process> & process_table) {}
 static void start_one_node_ssh(nodetab_process & p) {}
 static void start_nodes_mpiexec() {}
 
-static void finish_set_nodes(std::vector<nodetab_process> & process_table, int start, int stop) {}
+static void finish_set_nodes(std::vector<nodetab_process> & process_table, int start, int stop, bool charmrun_exiting=false) {}
```

Comment:
I think having the `=false` here is a compilation error.

Reply:
Good catch. I hadn't tested this on Windows.